### DISCUSSION TITLE
Connection Terminated [Jun 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6002] [Discord Threads]
> 🙋‍♂️ **unkn**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### can you share more about your db? is it hosting on cloud? or r u using heroku?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1111063744366248038)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
